docs/getting-started.md

```diff
@@ -215,6 +215,7 @@ for some languages:
 - [Google Go](https://github.com/ldmberman/GoEV3)
 - [Node.js](https://github.com/WasabiFan/ev3dev-NodeJS)
 - [Python](https://github.com/topikachu/python-ev3)
+- [C](https://github.com/theZiz/ev3c)
```

Comment:
On second though, maybe we should replace this list with a link to the other (libraries) page so that we don't have to maintain 2 lists of the same information.